src/main/java/frc/team4909/robot/Robot.java

```diff
@@ -107,14 +170,36 @@ public void autonomousPeriodic() {
    * This function is called periodically during operator control.
    */
   @Override
-  public void teleopPeriodic() {
-    System.out.println("Distance is  " + lidar.getDistance()); // Remove for competition (necessary only for testing)
+  public void teleopPeriodic() {  
+    System.out.println("Lidar value is: " + lidar.getDistance()); // Remove for competition (necessary only for testing)
+    StiltsStop = SmartDashboard.getBoolean("Stop Stilts", false);
+    if (StiltsStop == true){
+      new StopExtend();
+      SmartDashboard.putBoolean("Stop Stilts", false);
+    }
+    CargoIntake = SmartDashboard.getBoolean("Cargo In", false);
+    if (CargoIntake == true){
+      new CargoIntakeIn();
+      SmartDashboard.putBoolean("Cargo In", false);
+    }
+
+    CargoOuttake = SmartDashboard.getBoolean("Cargo Out", false);
+    if (CargoOuttake == true){
+      new CargoIntakeOut();
+      SmartDashboard.putBoolean("Cargo Out", false);
+    }
```

Comment:
what are these SmartDashboard references for? @ashwinc12

Reply:
is this intended to be a way to run these commands from smartdashboard? this should only construct the commands, it doesn't appear to be passing it to the scheduler